src/itemrenderer.cpp

```diff
@@ -9,6 +10,17 @@
 
 namespace newsboat {
 
+bool should_render_as_html(const std::string mime_type)
+{
+	std::set<std::string> html_mime_types = {
```

Comment:
Would `static` be appropriate here to prevent the set being constructed on each call?

Reply:
Good idea, amended into https://github.com/newsboat/newsboat/pull/1500/commits/81e72c3071bbb0fa14d288f87cebf988413f7c01